script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav__menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
```

Comment:
- [ ] Great job working on the functionality here so far but I couldn't see any menu in the mobile version size I think you put it on size 726px but this is for the desktop or tab version so kindly add it on small sizes so I can press on it and then it will open for me the menu so kindly add an icon for it so I can be able to see the menu properly!! ✅💪🏻💯💯

### Please remember that the minimum mobile screen size should be 375px.but in your design it start at 574px!💯💯📌♻♻♻
- Your design
![image](https://user-images.githubusercontent.com/58553711/215625533-16ba575a-d7ba-4f44-a1b2-caf2bd099a56.png)

- it should be 
![image](https://user-images.githubusercontent.com/58553711/215625826-1c705f49-5063-4062-a22b-a7bad41c4704.png)